app/src/main/java/chat/rocket/android/authentication/presentation/AuthenticationPresenter.kt

```diff
@@ -23,6 +23,10 @@ class AuthenticationPresenter @Inject constructor(
     private val serverInteractor: GetConnectingServerInteractor
 ) {
 
+    companion object {
+       val defaultServer = "https://open.rocket.chat"
```

Comment:
I think this variable should be ```defaultTestServer``` so that it's use case is clear.  Can you put this in the test config and not in the actual presenter?  We should have all test configs in one place.

I tested this with our server at dev.veranda.viasat.io and the first set of tests used our server, but then a whole bunch of other tests continued to use open.rocket.chat.  Can you test that way and fix please?

Reply:
Sure @ear-dev 👍

Reply:
So should I create a new config file for main package too in which we can put defaultTestServer variable as we do for test package(UnitTestConfig) and androidTest(UITestConfig) package? 
We can't access the variable for main package class from test package class as they have the same package name.
So For proper working of tests, we have to change that and those present in tests config files according to server requirement.
https://stackoverflow.com/questions/11027100/cannot-access-class-in-test-package-from-main-package-eclipse

Reply:
Oh shoot.......  thinking......  We need to have it all in one place.  Is there some other location where a single config file can be reached from both packages?  Maybe create a testConfig package?